configs/sot/siamese_rpn/siamese_rpn_r50_1x_otb.py

```diff
@@ -0,0 +1,51 @@
+_base_ = ['./siamese_rpn_r50_1x_lasot.py']
+
+crop_size = 511
+exemplar_size = 127
+search_size = 255
+
+# model settings
+model = dict(
+    head=dict(weighted_sum=False),
+    test_cfg=dict(rpn=dict(penalty_k=0.3, window_influence=0.5, lr=0.35)))
+
+data_root = 'data/'
+train_pipeline = [
```

Comment:
you need to pass the `train_pipeline` into `data`, otherwise, the `data` still uses the `train_pipeline` from lasot